testutils/src/main/scala/za/co/absa/testutils/CmdConfig.scala

```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  */
 
-package za.co.absa.enceladus.standardization
+package za.co.absa.testutils
```

Comment:
package should be `za.co.absa.enceladus.testutils` as all enceladus submodules follow this pattern `za.co.absa.enceladus.<submodule>`